web-ui/src/main/resources/catalog/locales/en-search.json

```diff
@@ -307,5 +308,8 @@
   "wpsDescribeProcessFailed": "DescribeProcess failed",
   "wpsExecute": "Execute",
   "wpsExecuteFailed": "Execute failed",
-  "wpsErrorCodeReturned": "The server returned status code"
+  "wpsErrorCodeReturned": "The serveur returned status code",
+  "mdvalid": "Metadata is valid",
+  "mdinvalid": "Metadata is invalid",
+  "mdnovalidation": "Metadata has no validation information."
```

Comment:
Maybe for the tooltip "Invalid metadata can't be publish" would be more meaningful ?